### PR TITLE
[DM-1101] New All devices view

### PR DESCRIPTION
**Pull request by bozhidardedov**

### CODE REVIEW

**Review by dafkk — APPROVED** [review]

File: content/release-10-15-0/whatsnew-10-15-0.md [hunk 1]
```diff
@@ -4,5 +4,10 @@ title: What´s new
 layout: bundle
 ---
 
-
 Release 10.15.0 includes the following new features or major feature enhancements.
+
+### New All devices view
```
Comment (BeateRixen) [inline content/release-10-15-0/whatsnew-10-15-0.md:9]: In the user documentation, we talk about "pages" and "tabs", the term "view" is rather technical. So it should be "All devices page" here. Also, I would not use the term "new" as the page is not new, better say "Modified" or "Improved".

File: content/release-10-15-0/whatsnew-10-15-0.md [hunk 2]
```diff
@@ -4,5 +4,10 @@ title: What´s new
...
+
+In an effort to provide users a more consistent experience across the platform the **All devices** view has been re-shaped. It is now implemented in the data grid fashion users know from **Subassets** and **Search** views. This offers improved appearence in various viewport sizes, easier filtering and sorting, pagination option (only for users with global inventory access) and sets solid ground for further improvements.
```
Comment (BeateRixen) [inline content/release-10-15-0/whatsnew-10-15-0.md:11]: users > you
**All devices** view > **All devices** page
users know > you already are familiar with
views > pages

**Comment by BeateRixen** [comment]
@bozhidardedov Has this also been modified in the User guide? 
Moreover, we might show the All devices page in a lot of places throughout the documentation, unfortunately probably with specific devices on it, depending on the use case. 
@janhommes Could you please provide a task for replacing all the screenshots? Most screenshots must probably be replaced by the respective teams which have the devices at place.

**Comment by bozhidardedov** [comment]
> @bozhidardedov Has this also been modified in the User guide? Moreover, we might show the All devices page in a lot of places throughout the documentation, unfortunately probably with specific devices on it, depending on the use case. @janhommes Could you please provide a task for replacing all the screenshots? Most screenshots must probably be replaced by the respective teams which have the devices at place.

I'm currently working on updating the User guide.

**Pull request merged**